src/components/text-block-header.ts

```diff
@@ -0,0 +1,8 @@
+import styled from "@emotion/styled";
+import { spacing, breakPoints } from "../constants";
+
+export const TextBlockHeader = styled("h3")`
+    max-width: ${breakPoints[3]};
+    margin: 0 auto ${spacing[0]} auto;
```

Comment:
same as in text-block.tsx, this can be written `margin: 0 auto ${spacing[0]};` for brevity

Reply:
Fixed, thanks again for your review